jib-core/build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
 
 dependencies {
   implementation "com.google.http-client:google-http-client:${dependencyVersions.GOOGLE_HTTP_CLIENT}"
+  implementation "com.google.http-client:google-http-client-apache-v2:${dependencyVersions.GOOGLE_HTTP_CLIENT_APACHE_V2}"
   implementation("com.google.auth:google-auth-library-oauth2-http:${dependencyVersions.GOOGLE_AUTH_LIBRARY_OAUTH2_HTTP}") {
```

Comment:
is this still overriding the value with some broken http client version?
Can we check via enforcement in the root build.gradle if this `exclude` is still required?

Reply:
actually I just saw we explicitly enforce this override anyway, perhaps we can take out the `excludes` and see what happens?

Reply:
I have been thinking we can remove this. Especially that we don't need to pin down the Apache components. With `excludes`, the dependency tree excludes `google-http-client` (of course),

```
+--- com.google.auth:google-auth-library-oauth2-http:0.18.0
|    +--- com.google.auto.value:auto-value-annotations:1.6.6
|    +--- com.google.code.findbugs:jsr305:3.0.2
|    +--- com.google.auth:google-auth-library-credentials:0.18.0
|    +--- com.google.http-client:google-http-client-jackson2:1.32.1
|    |    \--- com.fasterxml.jackson.core:jackson-core:2.9.9 -> 2.9.10
```
and without `excludes`, no problem as well since Gradle will upgrade its `google-http-client` to 1.33.0. (BTW, `google-auth-library-oauth2-http:0.18.0` is the latest we can get as of now.
```
+--- com.google.auth:google-auth-library-oauth2-http:0.18.0
|    +--- com.google.auto.value:auto-value-annotations:1.6.6
|    +--- com.google.code.findbugs:jsr305:3.0.2
|    +--- com.google.auth:google-auth-library-credentials:0.18.0
|    +--- com.google.http-client:google-http-client:1.32.1 -> 1.33.0 (*)
|    +--- com.google.http-client:google-http-client-jackson2:1.32.1
|    |    +--- com.google.http-client:google-http-client:1.32.1 -> 1.33.0 (*)
|    |    \--- com.fasterxml.jackson.core:jackson-core:2.9.9 -> 2.9.10
```

So I agree we should be good. And we have the check in the root `build.gradle` to ensure the version is correct.